$OpenBSD: patch-configure,v 1.25 2016/03/16 07:10:57 ajacoutot Exp $

- Chunk 1: GNOME Bug 627126 - gsettings schema files don't get installed on FreeBSD
glib commit 136e705e8383ff8848e425ac01278102d7badc52

- Chunk 2 : no librt on OpenBSD
https://bugs.webkit.org/show_bug.cgi?id=129973

- Chunk 3 : Don't force the use of -O2, the ports infrastructure
takes care of that

--- configure.orig	Mon Mar 14 08:37:02 2016
+++ configure	Tue Mar 15 08:11:20 2016
@@ -18338,26 +18338,14 @@ gsettings__enum_file = $(addsuffix .enums.xml,$(gsetti
 
 all-am: $(gsettings_SCHEMAS:.xml=.valid)
 uninstall-am: uninstall-gsettings-schemas
-install-data-am: install-gsettings-schemas
+install-data-am:
 
 .SECONDARY: $(gsettings_SCHEMAS)
 
-gsettings__base_list = \
-  sed "$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;s/\n/ /g" | \
-  sed "$$!N;$$!N;$$!N;$$!N;s/\n/ /g"
-
-install-gsettings-schemas: $(gsettings_SCHEMAS:.xml=.valid) $(gsettings__enum_file)
+install-gsettings-schemas: $(gsettings_SCHEMAS) $(gsettings__enum_file)
 	@$(NORMAL_INSTALL)
 	test -z "$(gsettingsschemadir)" || $(MKDIR_P) "$(DESTDIR)$(gsettingsschemadir)"
-	@list='\''$(gsettings__enum_file) $(gsettings_SCHEMAS)'\''; test -n "$(gsettingsschemadir)" || list=; \
-	for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  echo "$$d$$p"; \
-	done | $(gsettings__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '\''$(DESTDIR)$(gsettingsschemadir)'\''"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(gsettingsschemadir)" || exit $$?; \
-	done
+	$(INSTALL_DATA) $^ "$(DESTDIR)$(gsettingsschemadir)"
 	test -n "$(GSETTINGS_DISABLE_SCHEMAS_COMPILE)$(DESTDIR)" || $(GLIB_COMPILE_SCHEMAS) $(gsettingsschemadir)
 
 uninstall-gsettings-schemas:
@@ -21366,7 +21354,7 @@ if test "$found_opengl" = "yes"; then
     fi
     if test "$enable_egl" = "yes"; then
         acceleration_description="$acceleration_description, egl"
-        OPENGL_LIBS="$OPENGL_LIBS -lEGL"
+        OPENGL_LIBS="$OPENGL_LIBS -lEGL -lgbm"
     fi
     if test "$enable_glx" = "yes"; then
         acceleration_description="$acceleration_description, glx"
@@ -21944,7 +21932,7 @@ $as_echo "$ac_cv_search_shm_open" >&6; }
 ac_res=$ac_cv_search_shm_open
 if test "$ac_res" != no; then :
   test "$ac_res" = "none required" || LIBS="$ac_res $LIBS"
-  SHM_LIBS="-lrt"
+  SHM_LIBS=""
 fi
 
 
@@ -22640,8 +22628,8 @@ fi
 
 # Add the appropriate 'O' level for optimized builds.
 if test "$enable_optimizations" = "yes"; then
-    CXXFLAGS="$CXXFLAGS -O2"
-    CFLAGS="$CFLAGS -O2"
+    CXXFLAGS="$CXXFLAGS"
+    CFLAGS="$CFLAGS"
 
     if test "$c_compiler" = "gcc"; then
         CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=2"
